mevislab.github.io/content/tutorials/visualization/pathtracer/pathtracerexample2.md

```diff
@@ -0,0 +1,114 @@
+
+![SoPathTracerMaterial](/images/tutorials/visualization/pathtracer/V6.2_8.png "SoPathTracerMaterial")
+
+## Visualize Bones
+
+Follow the same steps for this scenario. Load this [Bones mask](/examples/visualization/example6/edited_Bones.mlimage) and examine it using `View2D`.  
+
+![Bones](/images/tutorials/visualization/pathtracer/V6.2_9.png "Bones")
+
+Start by refining the transparency of the cartilage structures using the `SoLUTEditor` to improve visibility and achieve a clearer depiction of the bones. Open it’s panel in the *Editor* tab, choose your *Index* number *3* and reduce the *Opacity* to *0*. 
+
+Alternatively, you can disable the toggle of the `SoPathTracerVolume` to make all structures disappear.
+
+
+![Bones](/images/tutorials/visualization/pathtracer/V6.2_10.png "Bones")
+
+Open the panel of `SoLUTEditor1` for the bones and make the following adjustments. Go to tab *Range* and set *New Range Min* to *0* and *New Range Max* to *127*.
+
+![SoLUTEditor1](/images/tutorials/visualization/pathtracer/V6.2_11_LUT_Bones.png "SoLUTEditor1")
+
+You can increase the *Shininess* of the bones and change the *Diffuse* color in the *Surface Brdf* tab within the `SoPathTracerMaterial1`.
+
+![SoPathTracerMaterial1](/images/tutorials/visualization/pathtracer/V6.2_SoPathTracerMaterial.png "SoPathTracerMaterial1")
+
+Enable the toggle of the `SoPathTracerVolume` to view both bones and structers. 
+
+![Result](/images/tutorials/visualization/pathtracer/V6.2_BonesResult.png "Result")
+
+## Visualize Vessels
+
+Repeat the process for the vessels. Load this [Vessels mask](/examples/visualization/example6/edited_Vessels.mlimage) and check it using `View2D`.
+
+![Vessels](/images/tutorials/visualization/pathtracer/V6.2_13.png "Vessels")
+
+Access the `SoLUTEditor2` panel for the vessels and modify the color settings within the *Editor* tab. Additionally, in the *Range tab*, set the *New Range Min* to *0* and the *New Range Max* to *255*.
+
+![SoLUTEditor2](/images/tutorials/visualization/pathtracer/V6.2_SoLUTEditor1_Vessels.png "SoLUTEditor2")
+
+![Final Result](/images/tutorials/visualization/pathtracer/V6.2_Final_Result.png "Final Result")
+
+## Summary:
+*	You can achieve impressive renderings using SoPathTracer and associated modules.
+* Render volumes efficiently in `SoPathTracer` scenes with `SoPathTracerVolume`, enabling diverse rendering options, LUT adjustments, and material enhancements.
+* Enhance your scene's look by adjusting materials and colors interactively using `SoPathTracerMaterial` and `SoLUTEditor`.
+* Use lighting modules such as `SoPathTracerAreaLight` and `SoPathTracerBackgroundLight` to optimize the illumination of your rendered scenes.
+
+{{< networkfile "examples/visualization/example6/pathtracer2.mlab" >}}
+
+[Bones mask](/examples/visualization/example6/LUT_Knee.xml)
+[Bones mask](/examples/visualization/example6/LUT_Bones.xml)
+[Bones mask](/examples/visualization/example6/LUT_Vessels.xml)
```

Comment:
Add the link to the LUT where it is needed. ONly the final mlab file should be here